tests/system/action/motion/test_create.py

```diff
@@ -290,6 +290,35 @@ def test_create_missing_origin_id(self) -> None:
             "data must not contain {'origin_id'} properties" in response.json["message"]
         )
 
+    def test_create_amendment(self) -> None:
+        self.set_models(self.permission_test_model)
+        self.set_models(
+            {
+                "meeting/1": {"name": "meeting_1 _test"},
+                "motion/3": {"meeting_id": 1, "category_id": 114, "block_id": 123},
+                "motion_category/12": {"meeting_id": 1},
+                "motion_block/13": {"meeting_id": 1},
+                "motion_category/114": {"meeting_id": 1},
+                "motion_block/123": {"meeting_id": 1},
+            }
+        )
+        response = self.request(
+            "motion.create",
+            {
+                "title": "test_Xcdfgee",
+                "meeting_id": 1,
+                "workflow_id": 12,
+                "text": "test",
+                "lead_motion_id": 3,
+                "block_id": 13,
+                "category_id": 12,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "motion/4", {"lead_motion_id": 3, "block_id": 123, "category_id": 114}
```

Comment:
expected: The block id and category id form the request is taken, so ids 12 and 13 since the request user has motion.can_manage

Reply:
Update the test_create and the test_set_number_mixin  tests.